seq2seq/third_party/trainers/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 The HuggingFace Team. All rights reserved.
```

Comment:
Did you write this file or copy it? If you wrote it, correct the copyright line. If you copied it (and potentially modified it) can you link me to where you copied it from? 

__init__.py files from the HF project might in some cases have to be totally replaced by us. If they are completely replaced and nothing was copied, we can treat them like any other code file that we write from scratch except, because they are special and need to be located in a specific place, we will allow them to go in third_party.

Reply:
I wrote it, thanks, done